integration/hyper_test.go

```diff
@@ -149,3 +149,16 @@ func (s *TestSuite) TestCreatePod(c *C) {
 	err = s.client.RemovePod(pod)
 	c.Assert(err, IsNil)
 }
+
+func (s *TestSuite) TestStartPod(c *C) {
+	podList, err := s.client.GetPodList()
+	c.Assert(err, IsNil)
+	c.Logf("Got PodList %v", podList)
+
+	if len(podList) == 0 {
+		return
+	}
+
+	err = s.client.StartPod(podList[0].PodID, "", "")
+	c.Assert(err, IsNil)
```

Comment:
@YaoZengzeng This test is not enough, you need to check the pod status becomes Running at least